app/Http/routes.js

```diff
@@ -15,6 +15,23 @@
 | Route.resource('user', 'UserController')
 */
 
-const Route = use('Route')
+const Route = use('Route');
+const Photo = use('App/Model/Photo');
 
 Route.on('/').render('welcome')
+
+Route.get('/photos', function * (req, res){
+  // const { url, caption } = yield req.session.all();
+  const { url, caption } = yield req.session.all();
```

Comment:
Fetch the photos from the database model not the session. In fact you can remove all of the session stuff.